src/core/src/sys/interactions.rs

```diff
@@ -443,4 +445,17 @@ mod test {
         interactions.add_global(Box::new(Wolf::new(1.0)));
         assert_eq!(interactions.globals().len(), 1);
     }
+
+    #[test]
+    fn test_globals_cutoff() {
```

Comment:
If we decide to make it a function of `Interactions`, what functionality would be good? Return the largest cutoff? Return a list of all cutoffs?